server/src/routes/api/v1/dogsRouter.js

```diff
@@ -14,6 +15,23 @@ dogsRouter.get("/", async (req, res) => {
   }
 });
 
+dogsRouter.post("/", async (req, res) => {
```

Comment:
we updated the endpoint here, make sure the post on the react side has been updated as well!